lib/IRGen/IRGenSIL.cpp

```diff
@@ -5456,6 +5456,12 @@ void IRGenSILFunction::visitCondFailInst(swift::CondFailInst *i) {
   llvm::BasicBlock *contBB = llvm::BasicBlock::Create(IGM.getLLVMContext());
   Builder.CreateCondBr(cond, failBB, contBB);
   Builder.emitBlock(failBB);
+  if (IGM.DebugInfo &&
```

Comment:
I think I would prefer sink the CodeView check into IRGenDebugInfo so IRGenSIL doesn't need to care about the debug info format.